### PR TITLE
doc: Update doc versions for 2.2.99-dev2 tag

### PR DESCRIPTION
**Pull request by umapraseeda**
Update doc versions for 2.2.99-dev2 tag

Signed-off-by: Uma Praseeda <uma.praseeda@nordicsemi.no>

### CODE REVIEW

**Review by nordicjm — APPROVED** [review]

**Review by peknis — APPROVED** [review]

**Review by gmarull — APPROVED** [review]

File: doc/nrf/releases/release-notes-changelog.rst [hunk 1]
```diff
@@ -86,7 +77,7 @@ Matter
   * Default support for nRF7002 revision B.
   * Specific QR code and onboarding information in the documentation for each :ref:`Matter sample <matter_samples>` and the :ref:`Matter weather station <matter_weather_station_app>`.
   * The Bluetooth LE advertising arbiter class that enables easier coexistence of application components that want to advertise their Bluetooth LE services.
-  * Support for erasing settings partition during DFU over Bluetooth LE SMP for the Nordic nRF52 Series' SoCs.
+  * Support for erasing settings partition during DFU over Bluetooth LE SMP for the Nordic Semiconductor's nRF52 Series' SoCs.
```
Comment (greg-fer) [inline doc/nrf/releases/release-notes-changelog.rst:80]:
```suggestion
  * Support for erasing settings partition during DFU over Bluetooth LE SMP for the Nordic nRF52 Series' SoCs.
```
The original text matches requirements from Marketing: https://projecttools.nordicsemi.no/confluence/display/MAR/Product+names

File: doc/nrf/releases/release-notes-changelog.rst [hunk 2]
```diff
@@ -95,7 +86,7 @@ Matter
   * :ref:`ug_matter_device_certification` page with a section about certification document templates.
   * :ref:`ug_matter_overview_commissioning` page with information about :ref:`ug_matter_network_topologies_commissioning_onboarding_formats`.
   * Default retry intervals used by Matter Reliability Protocol for Matter over Thread to account for longer round-trip times in Thread networks with multiple intermediate nodes.
-  * Changed Bluetooth LE connection timeout and parameters update timeout parameters to make communication over Bluetooth LE more reliable.
+  * The Bluetooth LE connection timeout parameters and the update timeout parameters have been changed to make communication over Bluetooth LE more reliable.
```
Comment (greg-fer) [inline doc/nrf/releases/release-notes-changelog.rst:89]:
```suggestion
  * The Bluetooth LE connection timeout parameters and the update timeout parameters to make communication over Bluetooth LE more reliable.
```

File: doc/nrf/releases/release-notes-changelog.rst [hunk 3]
```diff
@@ -346,7 +337,7 @@ Peripheral samples
 
   * Added support for the nRF7002 board.
   * Fixed sample building with support for the Skyworks front-end module.
-  * Updated documentation to clarify that this sample is dedicated for the short-range radio (Bluetooth LE, IEEE 802.15.4 and proprietary modes).
+  * Updated the documentation to clarify that this sample is dedicated for the short-range radio (Bluetooth LE, IEEE 802.15.4 and proprietary modes).
```
Comment (greg-fer) [inline doc/nrf/releases/release-notes-changelog.rst:340]:
```suggestion
  * Updated the documentation to clarify that this sample is dedicated for the short-range radio (Bluetooth LE, IEEE 802.15.4, and proprietary modes).
```

**Review by greg-fer — APPROVED** [review]
One nitpick.

**Review by bama-nordic — APPROVED** [review]
LGTM

**Review by b-gent — APPROVED** [review]

**Pull request merged**